ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleManager.java

```diff
@@ -133,11 +131,17 @@ VungleNativeAd getVungleNativeAd(String placement, AdConfig adConfig, VungleList
         return bannerAd;
     }
 
-    void removeActiveBanner(String placementId) {
+    void removeActiveBanner(String placementId, VungleNativeAd willRemoveBannerAd) {
         if (placementId == null) {
             return;
         }
-        activeBannerAds.remove(placementId);
+        Log.d(TAG, "removeActiveBanner");
+        VungleNativeAd activeBannerAd = activeBannerAds.get(placementId);
+        // must make sure that will be removed active banner ad is what you want.
+        if (activeBannerAd == willRemoveBannerAd) {
```

Comment:
The key solution. AdMob `onDestroy()` can be called before or after the next requestBannerAd for the same placementId(refresh) and out of our control. If invoked after refresh and new banner is created, remove activeBannerAds will cause the previous created banner ad instance doesn't be maintained in activeBannerAds no longer. So we will catch the exeption `Playing or Loading operation ongoing` when the next refresh.

Reply:
I can remember the bug that onDestroy call be not called at all depending on device/OS.
Can we still finish & clean old banner on creating a new one?

Reply:
a similar fix was already done in https://github.com/Vungle/publisher-admob-android-plugin/blob/6.4.x-release/adapter/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java#L178
@palrahul @HuFangshuai  why we had lost that code?

Reply:
@alexander-gubov thanks for your feedback. These adapterId related code removed for AdMob recommendation. But after that we realized it's required. I had discussed with Rahul about this issue last week and I submit this PR for another solution purpose without using adapterId.